valmlord/gulpify
# feat: add pug template engine

## Description
This pull request integrates the Pug template engine into the Gulp boilerplate, allowing for the use of Pug templates in the project.

## Changes
- Added `gulp-pug` package to `package.json`.
- Updated `gulpfile.js` to include a new task for compiling Pug templates to HTML.
- Created a `pug` directory for Pug files.

## New Gulp Tasks
- `pug`: Compiles Pug templates from the `app/views` directory to HTML files in the `build` directory.

## How to Test
1. Place a Pug template file in the `app/views` directory.
2. Execute the `gulp pug` command.
3. Verify that the Pug template is compiled to an HTML file in the `build` directory.

## Additional Notes
- This update aims to enhance the development process by enabling the use of Pug templates for more efficient HTML generation.
